src/components/Dashboard/index.jsx

```diff
@@ -0,0 +1,135 @@
+import React from 'react';
+import './styles/dashboard.scss';
+
+//All the various cells I'm using have their own js files
+import TwitterFeed from './cells/twitter';
+import CountDown from './cells/countdown';
+import Time from './cells/time';
+import Logo from './cells/logo';
+import Sponsors from './cells/sponsors';
+
+
+class SplitColumn extends React.Component {
```

Comment:
I feel like the only components you need are `Dashboard` and `Cell`. Does split column refer to one column of three cells?